execution/src/vm.rs

```diff
@@ -255,9 +314,9 @@ impl VM {
                 let program = instance
                     .exports
                     .get_function("main")
-                    .unwrap()
+                    .unwrap() // TODO do not unwrap !
```

Comment:
Why do not unwrap ?

Reply:
because we don't whant the whole node to crash if the "main" function is missing in some random bytecode

Reply:
So as usual we don't want any unwrap unless it's guaranteed not to panic, this one is not special ?

Reply:
same rule yeah, but here it's even worse: it's an unchecked user-entered input. Having an unwrap here is basically suicide in terms of security